project/Dependencies.scala

```diff
@@ -1,11 +1,13 @@
 import sbt._
 
 object Dependencies {
-  val zioVersion = "2.0.0-M4"
+
+  val zioVersion = "2.0.0-RC1"
   lazy val deps  = Seq(
     "dev.zio"                %% "zio-streams"                 % zioVersion,
     "dev.zio"                %% "zio-test"                    % zioVersion % Test,
     "dev.zio"                %% "zio-test-sbt"                % zioVersion % Test,
+    "dev.zio"                %% "zio-config"                  % "1.0.10",
```

Comment:
I don't think this is used

Reply:
True I was planning on using it for other code for loading config , just didn't want to push that info this commit .
I'll remove it